blocks/library/freeform/index.js

```diff
@@ -5,11 +5,9 @@ import { registerBlock, query, setUnknownTypeHandler } from 'api';
 
 const { html } = query;
 
-registerBlock( 'core/freeform', {
+const blockSettings = {
```

Comment:
Actually, I exported only the one that I tested for now but thinking we export the object as default (including the slug). see discussion here https://github.com/WordPress/gutenberg/pull/448#discussion_r112205478